includes/class-pubsubhubbub-admin.php

```diff
@@ -38,7 +38,7 @@ public static function register_settings() {
 				'type'              => 'string',
 				'description'       => __( 'The WebSub/PubSubHubbub endpoints', 'pubsubhubbub' ),
 				'show_in_rest'      => true,
-				'default'           => "https://pubsubhubbub.appspot.com\nhttps://pubsubhubbub.superfeedr.com\nhttps://websubhub.com/hub",
+				'default'           => implode( PHP_EOL, Pubsubhubbub::DEFAULT_HUBS ),
```

Comment:
I thought this might be enough! Isn't it?

Reply:
Seems to be more than enough once a base option is set in the DB. It correctly updates the value once it's first saved, but fails to actually create the option if none is found from what I'm seeing...